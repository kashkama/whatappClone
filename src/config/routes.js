import Home from "./../screens/home";
import ChatScreen from "./../screens/chatScreen";

export const routes = {
    Home: {screen: Home},
    Chat: {screen: ChatScreen}
};
