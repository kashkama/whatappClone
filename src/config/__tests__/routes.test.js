import React from "react";
import {routes} from "./../routes";
import Home from "./../../screens/home";
import ChatScreen from "./../../screens/chatScreen";

it("has all needed routes", () => {
    expect(routes).toMatchSnapshot();
})

it("has home screen", () => {
    expect(routes.Home.screen).toBe(Home);
})

it("has chart screen", () => {
    expect(routes.Chat.screen).toBe(ChatScreen);
})