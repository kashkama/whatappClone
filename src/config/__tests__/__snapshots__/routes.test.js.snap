// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has all needed routes 1`] = `
Object {
  "Chat": Object {
    "screen": [Function],
  },
  "Home": Object {
    "screen": [Function],
  },
}
`;
